great_expectations/data_context/data_context.py

```diff
@@ -107,24 +102,57 @@ def create(cls, project_root_dir=None):
         Returns:
             DataContext
         """
+
         if not os.path.isdir(project_root_dir):
-            raise ge_exceptions.DataContextError("project_root_dir must be a directory in which to initialize a "
-                                                 "new DataContext")
-        else:
-            try:
-                os.mkdir(os.path.join(project_root_dir, "great_expectations"))
-            except (FileExistsError, OSError):
-                raise ge_exceptions.DataContextError(
-                    "Cannot create a DataContext object when a great_expectations directory "
-                    "already exists at the provided root directory.")
+            raise ge_exceptions.DataContextError(
+                "The project_root_dir must be an existing directory in which "
+                "to initialize a new DataContext"
+            )
+
+        ge_dir = os.path.join(project_root_dir, "great_expectations")
+        safe_mmkdir(ge_dir, exist_ok=True)
+        cls.scaffold_directories(ge_dir)
 
+        if os.path.isfile(os.path.join(ge_dir, "great_expectations.yml")):
+            message = """Warning. An existing `great_expectations.yml` was found here: {}.
```

Comment:
I'm torn...based on the above, I think these are maybe just info logs vs. warnings, since we're using this behavior during onboarding of teammembers, right?